lib/cn/cn.c

```diff
@@ -1044,10 +1053,10 @@ cn_open(
     }
 
     log_info(
-        "%s/%s cnid %lu fanout %u pfx_len %u vcomp %u,%u"
+        "%s/%s cnid %lu pfx_len %u vcomp %u,%u"
```

Comment:
I'd start this log entry with "opening kvs %s" so anybody that stumbles across it in a log knows it has good info on the kvs at open time.  Also nice for grepping logs.

Reply:
FWIW, the log message will be preceded with "cn_open:"....

Reply:
Done.